### DISCUSSION TITLE
start recording automatically when a window show up

### DISCUSSION THREAD

**Discussion by fenchu** [obsproject/obs-studio#5041] [General]
Hi I'm pretty new to this software, and recording manually in any format and framerate works excellent.

But I like it to automatically start recording each time google-chrome pops up a window, and then stop recording when it closes.
and just increment the *.mkv file with the next window.

I see the chrome.exe in capture windows, but it is not updating so you need to close and choose new capture to get window list updated. so I'm unable to record the first seconds of the window.

I can leave a capture on for the entire screen but the files become large and 90% has no interest.
I like 30fps and small window to capture rendering artifacts we do not see with selenium.

Thanks for a great product.

**Comment by dodgepong** [comment]
Please post feature requests on our ideas page: https://ideas.obsproject.com
